modules/gdscript/gdscript_parser.h

```diff
@@ -733,10 +733,9 @@ class GDScriptParser {
 		Multiplayer::RPCConfig rpc_config;
 		MethodInfo info;
 		LambdaNode *source_lambda = nullptr;
-#ifdef TOOLS_ENABLED
+
 		Vector<Variant> default_arg_values;
 		String doc_description;
```

Comment:
I think this can be kept tools only, no?

Reply:
The `p_function->default_arg_values.size()` in gdscript_analyzer.cpp is protected by a is debug.

Reply:
Maybe this should be a is debug defined too

Reply:
The proper fix is likely to leave this unchanged and make *this* `TOOLS_ENABLED`, like all other code accessing `default_arg_values`: https://github.com/godotengine/godot/pull/58835/files#diff-e01550d654cd3b28e2c1abaa393c95a08008ea22c5860cbd034b515b84635237R1157

Reply:
This change I made looks ok.